webapp/src/index.tsx

```diff
@@ -635,12 +631,17 @@ export default class Plugin {
             );
         };
 
-        const fetchChannels = async (): Promise<AnyAction[]> => {
+        const fetchChannels = async (skipChannelID?: string): Promise<AnyAction[]> => {
             const actions = [];
             try {
                 const data = await RestClient.fetch<CallChannelState[]>(`${getPluginPath()}/channels`, {method: 'get'});
 
                 for (let i = 0; i < data.length; i++) {
+                    if (skipChannelID === data[i].channel_id) {
+                        logDebug('skipping channel from state loading', skipChannelID);
+                        continue;
+                    }
```

Comment:
This is important as it will avoid messing the state for the current call which should only come from websocket from now on.

Reply:
Can we add a comment in the code to remind us about the what and why? I can imagine forgetting and then spending time trying to reason through it.